packages/evm/src/eof.ts

```diff
@@ -62,33 +64,97 @@ export const codeAnalysis = (container: Buffer) => {
   return sectionSizes
 }
 
-export const validOpcodes = (code: Buffer) => {
+export const validOpcodes = (code: Buffer, common: Common) => {
```

Comment:
Hmm, I am not sure if this method might have made it in the half-official part of the API and people use this in the wild. 🤔 Maybe we should think of an alternative way to add a second required parameter.

Maybe we take an optional parameter `common` and then do `common?.isActivatedEIP(4200)`? Might have less overall side effects.

Reply:
I honestly don't think people are using this out in the wild, but I might be wrong. I can indeed make it optional.

Reply:
Made it optional (this also fixes our API tests)

Reply:
Haha, are we counting as "the wild" as well? 😜